tests/test.js

```diff
@@ -0,0 +1,39 @@
+const assert = require('assert')
+const spawn = require('child_process').spawn
+const uuidV4 = require("uuid/v4")
+const fs = require('fs')
+
+describe('WebUi Generator', () => {
+  let uniqueId = uuidV4()
+  let email = 'fossasia@gmail.com'
+  let args = [
+    "-g", "https://github.com/fossasia/yaydoc.git",
```

Comment:
yeah, it does. here is the output
```
  WebUi Generator
    ✓ exit code shoul be zero
    ✓ preview folder should be created
    ✓ zipped documentation should be created
  3 passing (2m)
``` 
i didn't add the `npm test` for this pr. once my previous pr got merged. This is good to go with travis